config/buildbuddy.release.yaml

```diff
@@ -1,10 +1,10 @@
 # N.B. This file is bundled and distributed in the enterprise docker image.
 database:
-  data_source: "sqlite3:///buildbuddy/buildbuddy.db"
+  data_source: "sqlite3:///app/server/cmd/buildbuddy/buildbuddy.runfiles/buildbuddy/buildbuddy.db"
 storage:
   ttl_seconds: 2592000 # 30 days
   disk:
-    root_directory: "/buildbuddy/blobs/"
+    root_directory: "/app/server/cmd/buildbuddy/buildbuddy.runfiles/buildbuddy/blobs/"
 cache:
   disk:
-    root_directory: "/buildbuddy/cache/"
```

Comment:
I think we `mkdir -p` on these directories, so they shouldn't need to exist in the image.

Is there some error we're seeing that you think this would fix?

Reply:
NVM, just read the linked issue.

I think maybe we should just `mkdir -p` on the sqlite parent dir.

Reply:
Swapped to using a symlink, as we discussed offline.